.github/workflows/CI.yml

```diff
@@ -52,10 +52,7 @@ jobs:
       - uses: julia-actions/julia-processcoverage@v1
       - uses: codecov/codecov-action@v5
         with:
-          file: lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
-          plugins: noop
-          disable_search: true
```

Comment:
Actually I think this line broke it, together with the fact that `file` is gone and one has to use `files`. So in this sense I am really hopefully this will work.

What mystifies me is why PR reports were working all along?!